src/FormModel.php

```diff
@@ -244,6 +236,11 @@ public function rules(): array
         return [];
     }
 
+    public function getRules(): array
+    {
+        return $this->rules();
```

Comment:
I want to rename form's `rules` to `getRules` instead of validator's `getRules` to `rules` :)

Reply:
???